projects/addon-commerce/enums/currency-code.ts

```diff
@@ -371,22 +379,14 @@ export enum TuiCurrencyCode {
      * @description Currency symbol: TMT
      */
 
+    NewTurkmenManat = `934`,
     /**
-     * Japanese yen
-     *
-     * @description Numeric code: 392
-     * @description Alphabetic code: JPY
-     * @description Countries and territories: Japan
-     * @description Currency symbol: ¥
-     */
-    Yen = `392`,
-    /**
-     * Chinese yuan
+     * Brazilian Real
      *
-     * @description Numeric code: 156
-     * @description Alphabetic code: CNY
-     * @description Countries and territories: China
-     * @description Currency symbol: CN¥
+     * @description Numeric code: 986
+     * @description Alphabetic code: BRL
+     * @description Countries and territories: Brazilia
+     * @description Currency symbol: R$
      */
-    YuanRenminbi = `156`,
+    BrazilianReal = `986`,
```

Comment:
Did you rollback the changes that messed it up? Maybe it's a good idea to keep it alphabetically sorted?

Reply:
fixed